visual-java/src/main/java/com/saucelabs/visual/VisualApi.java

```diff
@@ -176,7 +177,8 @@ public VisualApi(
               + "Please check your SauceLabs username and access key at https://app.saucelabs.com/user-settings");
     }
     this.client = new GraphQLClient(url, username, accessKey);
-    this.sessionId = driver.getSessionId().toString();
```

Comment:
Shouldn't we keep this one too. We always require a string, while driver provides a `SessionId`.

IMO It will be more readable to have `this.sessionId` rather than `this.driver.getSessionId().toString()`

WDYT?

Reply:
Yeah I was not sure about removing it either. Added it.